src/prefect/tasks/soda/soda.py

```diff
@@ -0,0 +1,79 @@
+from sodasql.scan.scan_builder import ScanBuilder
```

Comment:
Since the underlying library is `sodasql` (and "Soda SQL" seems to be a term that soda.io wants users to use), should the task module be named `prefect.tasks.sodasql` instead of `prefect.tasks.soda`?